src/main/java/core/basesyntax/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private final Storage storage;
+
+    public ReturnOperation(Storage storage) {
+        this.storage = storage;
+    }
+
+    @Override
+    public void count(FruitTransaction transaction) {
+        if (transaction.getQuantity() < 0) {
+            throw new RuntimeException("Quantity can't be less than 0");
+        }
+        storage.addFruit(transaction.getFruit(), transaction.getQuantity());

```

Comment:
The `count` method name is not descriptive of its function. Consider renaming it to something more intuitive, such as `applyReturnTransaction` or `handleReturn`, to clearly convey its purpose.